# Máquinas de corriente contínua

## Ejemplo 7-1
La figura muestra una espira giratoria sencilla entre caras polares conectadas a una batería y a un resistor mediante un interruptor. El resistor que se ilustra representa la resistencia total de la batería y del alambre de la máquina. Las dimensiones físicas y características de la máquina son:

$$\begin{array}{ll}
r = 0,5 \, \mathrm{m} & l = 1,0 \, \mathrm{m} \\
R = 0,3 \, \Omega & B = 0,25 \, \mathrm{T} \\
V_B = 120 \, \mathrm{V}
\end{array}$$

<img src="img/fig_7_1.svg" width=600>

1. ¿Qué sucede cuando se cierra el interruptor?
2. ¿Cuál es la corriente de arranque máxima de la máquina? ¿Cuál es su velocidad angular en estado estacionario en vacío?
3. Suponga que se añade una carga a la espira y el par resultante es $10 \, \mathrm{N \cdot m}$. ¿Cuál será la nueva velocidad en estado estacionario? ¿Cuánta potencia se suministrara al eje de la máquina? ¿Cuánt potencia suministrará la batería? ¿Esta máquina actúa como motor o como generador?
4. Suponga que la máquina está en vacío de nuevo y se aplica al eje un par de $7,5 \, \mathrm{N \cdot m}$ en el sentido de la rotación. ¿Cuál es la nueva velocidd de estado estacionario? ¿Esta máquina es un motor o un generador?
5. Suponga que la máquina funciona en vacío. ¿Cuál será la velocidad final en estado estacionario si se reduce la densidad de flujo a $0,20 \, \mathrm{T}$?

### Solución
1. Cuando se cierra el interruptor de la figura, fluye corriente en la espira. Puesto que la espira está estacionaria inicialmente, $e_\mathrm{ind} = 0$. Por lo tanto, la corriente está dada por

$$ i = \frac{V_B - e_\mathrm{ind}}{R} = \frac{V_B}{R} $$

La corriente fluye a través de la espira del rotor y produce un par

$$ \tau_\mathrm{ind} = \frac{2}{\pi} \phi i \qquad \mbox{(antihorario)}$$

Este par inducido produce una aceleración angular en dirección contraria a la de las manecillas del reloj, por lo que el rotor de la máquina comienza a girar. Pero conforme el rotor gira, se produce un voltaje inducido en el motor. Este voltaje inducido está dado por

$$ e_\mathrm{ind} = \frac{2}{\pi} \phi \omega_m $$

por lo que la corriente $i$. Conforme cae la corriente, disminuye $\tau_\mathrm{ind} = (2/\pi) \phi i \, \downarrow$ y la máquina gira en estado estacionario con $\tau_\mathrm{ind} = 0$, y el voltaje de la batería $V_B = e_\mathrm{ind}$.

2. En condiciones de arranque, la corriente de la máquina es

$$ i = \frac{V_B}{R} $$

In [2]:
# Datos:
r = 0.5 # [m]
R = 0.3 # [ohm]
V_B = 120 # [V]
l = 1.0 # [m]
B = 0.25 # [T]

In [3]:
i = V_B/R
print("i = %.1f [A]"%i)

i = 400.0 [A]


En condiciones de estado estacionario en vacío, el par inducido $\tau_\mathrm{ind}$ debe ser cero. Pero si $\tau_\mathrm{imd} = 0$, esto implica que la corriente $i$ debe ser igual a cero, puesto que $\tau_\mathrm{ind} = (2/\pi) \phi i$ y el flujo es distinto de cero. El hecho de que $i = 0 \, \mathrm{A}$ significa que el voltaje de la batería $V_B = e_\mathrm{ind}$. Por lo tanto, la velocidad del rotor es

$$ V_B = e_\mathrm{ind} = \frac{2}{\pi} \phi \omega_m $$

$$ \omega_m = \frac{V_B}{(2/\pi) \phi} = \frac{V_B}{2rlB} $$

In [4]:
w_m = V_B/(2*r*l*B)

In [5]:
print("w_m = %.1f [rad/s]"%w_m)

w_m = 480.0 [rad/s]


3. Si se aplica un par de carga de $10 \, \mathrm{N \cdot m}$ al de eje de la máquina, comenzará a perder velocidad. Pero conforme $\omega_m$ disminuye, se reduce $e_\mathrm{ind} = (2/\pi) \phi \omega_m \, \downarrow$ y la corriente del rotor se incrementa $[i = V_B - e_\mathrm{ind} \downarrow)/R]$. A medida que la corriente del rotor se incrementa, $| \tau_\mathrm{ind}|$ también aumenta, hasta que $|\tau_\mathrm{ind} | = | \tau_\mathrm{carga} |$ a una velocidad $\omega_m$ menor.

En estado estacionario, $|\tau_\mathrm{carga}| = |\tau_\mathrm{ind}| = (2/\pi) \phi i$. Por lo tanto,

$$ i = \frac{\tau_\mathrm{ind}}{(2/\pi) \phi} = \frac{\tau_\mathrm{ind}}{2rlB} $$

In [7]:
tau_ind = 10 # [N·m]

i = tau_ind/(2*r*l*B)

In [8]:
print("i = %.1f [A]"%i)

i = 40.0 [A]


Por la ley de voltaje de Kirchhoff, $e_\mathrm{ind} = V_B - iR$, por lo que

In [10]:
e_ind = V_B - i*R

In [11]:
print("e_ind = %.1f [V]"%e_ind)

e_ind = 108.0 [V]


Finalmente, la velocidad del eje es
$$ \omega_m = \frac{e_\mathrm{ind}}{(2/\pi) \phi} = \frac{e_\mathrm{ind}}{2rlB} $$

In [12]:
w_m = e_ind/(2*r*l*B)

In [13]:
print("w_m = %.1f [rad/s]"%w_m)

w_m = 432.0 [rad/s]


La potencia suministrada al eje es
$$ P = V_B i $$

In [14]:
P = V_B*i

In [15]:
print("P = %.0f [W]"%P)

P = 4800 [W]


Esta máquina opera como motor, es decir, convierte potencia eléctrica en potencia mecánica.

Si se aplica un par en la dirección del movimiento, el rotor acelera. Conforme se incrementa la velocidad, aumenta el voltaje interno $e_{ind}$ y sobrepasa $V_B$, por lo que la corriente fluye hacia fuera de la parte superior de la barra y hacia la batería. Esta máquina ahora actúa como generador. La corriente provoca un par inducido opuesto a la dirección del movimiento. El par inducido se opone al par externo aplicado y a la carga $|\tau_\mathrm{carga}| = |\tau_\mathrm{ind}|$ a una velocidad $\omega_m$.

La corriente en el rotor será

$$ i = \frac{\tau_\mathrm{ind}}{(2/\pi) \phi} = \frac{\tau_\mathrm{ind}}{2rlB} $$

In [16]:
tau_ind = 7.5 # [N·m]

In [17]:
i = tau_ind/(2*r*l*B)

In [18]:
print("i = %.1f [A]"%i)

i = 30.0 [A]


El voltaje inducido $e_\mathrm{ind}$ es
$$ e_\mathrm{ind} = V_B + i R $$

In [19]:
e_ind = V_B + i*R

In [20]:
print("e_ind = %.1f [V]"%e_ind)

e_ind = 129.0 [V]


Finalmente, la velocidad del eje es

$$ \omega_m = \frac{e_\mathrm{ind}}{(2/\pi) \phi} = \frac{e_\mathrm{ind}}{2rlB} $$

In [21]:
w_m = e_ind/(2*r*l*B)

In [22]:
print("w_m = %.1f [rad/s]"%w_m)

w_m = 516.0 [rad/s]


4. Puesto que inicialmente, con las condiciones propuestas al principio, la máquina no tiene carg, la velocidad $\omega_m = 480 \, \mathrm{rad/s}$. Si disminuye el flujo, se presenta un transitorio. Sin embargo, una vez que se termina el transitorio, la máquina deba tener una vez más un par igual a cero puesto que aún no hay carga en el eje. Si $\tau_\mathrm{ind} = 0$, entonces la corriente en el rotor debe ser cero y $V_B = e_\mathrm{ind}$. Entonces, la velocidad en el eje es

$$ \omega_m = \frac{e_\mathrm{ind}}{(2/\pi) \phi} = \frac{e_\mathrm{ind}}{2rlB} $$

In [28]:
e_ind = 120 # [V]
B = 0.20 # [T]

In [29]:
w_m = e_ind/(2*r*l*B)

In [30]:
print("w_m = %.1f [rad/s]"%w_m)

w_m = 600.0 [rad/s]


*Nótese que cuando el flujo en la máquina disminuye, la velocidad se incrementa.* Éste es el mismo comportamiento que se observa en una máquina lineal y es la misma manera en que se comportan los motores de cd reales.